docs/podman-create.1.md

```diff
@@ -727,11 +727,13 @@ The followings examples are all valid:
 Without this argument the command will be run as root in the container.
 
 **--userns**=host
+**--userns**=keep-id
 **--userns**=ns:my_namespace
 
-Set the user namespace mode for the container. The use of userns is disabled by default.
+Set the user namespace mode for the container.  It defaults to the **PODMAN_USERNS** environment variable.  An empty value means user namespaces are disabled.
 
 - `host`: run in the user namespace of the caller. This is the default if no user namespace options are set. The processes running in the container will have the same privileges on the host as any other process launched by the calling user.
+- `keep-id`: creates a user namespace where the current rootless user is mapped to its same UID:GID.  It is ignored for root containers.
```

Comment:
suggest:
`keep-id`: creates a user namespace where the current rootless user's UID:GID are mapped to the same values in the container.  This option is ignored for containers created by the root user.